### PR TITLE
update according to new interface

### PR DESCRIPTION
**Pull request by pavel-shermayster-sage**

### CODE REVIEW

**Review by codelicous — COMMENTED** [review]
remove unused imports

File: src/app/page.tsx [hunk 1]
```diff
@@ -6,71 +6,182 @@ import Avatar3Image from "@/assets/images/avatars/avatar-3.png";
...
+const getUsers = async () => {
+  return Promise.resolve(users);
+};
+
+const getEntries = async () => {
+  return Promise.resolve(entries);
+};
+
+const getGameData = async () => {
+  const usersMap = (await getUsers()).reduce(
```
Comment (codelicous) [inline src/app/page.tsx]: i suggest dedicating the reduce and the map callback to have a  dedication declared function for the sake of declarations and shortening the function

Comment (pavel-shermayster-sage) [reply src/app/page.tsx]: I've abstracted reduce, but if take away entries creation, there is no poin in getGameData function

Comment (codelicous) [reply src/app/page.tsx]: all i meant was that rather then having inline logic passing , better to have a named function passed to the reduce callback

any way not so important

**Pull request merged**